dashboard/common/db_util.py

```diff
@@ -117,7 +120,7 @@ def get_last_cron_run():
     return datetime.min
 
 
-def get_canvas_data_date():
+def get_canvas_data_date() -> datetime:
```

Comment:
Do we need to import `datetime` if we're going to reference the object type here?

Reply:
It looks like it's already imported on line 5

Reply:
Doh, sorry, I missed that, GitHub was hiding it from me.